indico/modules/events/controllers/creation.py

```diff
@@ -49,22 +50,40 @@ def _default_category(self):
         else:
             return Category.get(category_id, is_deleted=False)
 
+    def _get_prepared_data(self):
+        try:
+            cache = GenericCache('event-preparation')
+            data = cache.get(request.args['event_uuid'])
+        except KeyError:
+            return None
+        if not data:
+            return None
+        title = data['title']
+        start_dt = data['start_dt']
+        end_dt = start_dt + relativedelta(minutes=data['duration'])
+        return start_dt, end_dt, title
+
     def _get_form_defaults(self):
         category = self._default_category
         tzinfo = timezone(config.DEFAULT_TIMEZONE)
         if category is not None:
             tzinfo = timezone(category.timezone)
 
-        # try to find good dates/times
-        now = now_utc(exact=False)
-        start_dt = now + relativedelta(hours=1, minute=0)
-        if start_dt.astimezone(tzinfo).time() > time(18):
-            start_dt = tzinfo.localize(datetime.combine(now.date() + relativedelta(days=1), time(9)))
-        end_dt = start_dt + relativedelta(hours=2)
+        prepared_data = self._get_prepared_data()
+        if prepared_data:
+            start_dt, end_dt, title = prepared_data
+        else:
+            # try to find good dates/times
```

Comment:
title is never set when you get in this block, so let's add `title = ''` here

Reply:
Thanks.